packages/office-ui-fabric-react/src/components/Button/ButtonThemes.ts

```diff
@@ -70,6 +70,10 @@ export function standardStyles(theme: ITheme): IButtonStyles {
               color: 'Highlight'
             }
           }
+        },
+        [HighContrastSelector]: {
+          border: `1px solid white`,
+          borderLeft: 'none'
```

Comment:
Why is this here? This is different from the styling in #10364.

Reply:
Hmm yeah I don't see this in the original fabric 7 change https://github.com/OfficeDev/office-ui-fabric-react/pull/10364/files#diff-83f447f02b4959069f9feb281f1c83b2R98

Reply:
In Fabric 7 a border is introduced to the button in non-HC mode. Since it's not there in Fab 6 i have to add this white border only in HC for there to be one.

Reply:
Slightly confused - why does this need to be added in Fabric 6 if the border is only expected for Fabric 7? There are screener changes down below so we should be careful not to break anyone still using Fabric 6.

Reply:
I think this is because in Fabric 6 we don't use a border on the `Button` due to the background color indicating where it ends. However, this background color is no longer visible in High Contrast Mode, which is why we need a border there.